clients/src/main/resources/common/message/ProduceResponse.json

```diff
@@ -27,7 +27,10 @@
   //
   // Version 5 added LogStartOffset to filter out spurious
   // OutOfOrderSequenceExceptions on the client.
-  "validVersions": "0-7",
+  //
+  // Version 8 added ErrorRecords and ErrorMessage to include information about
```

Comment:
Can you add some test cases to `MessageTest` to verify expected default/ignorable behavior?